valhalla/baldr/graphtileheader.h

```diff
@@ -162,15 +163,18 @@ class GraphTileHeader {
    * Get the base (SW corner) of the tile.
    * @return Returns the base lat,lon of the tile (degrees).
    */
-  const midgard::PointLL& base_ll() const {
-    return static_cast<const midgard::PointLL&>(base_ll_);
+  midgard::PointLL base_ll() const {
+    GraphId id(graphid_);
+    return TileHierarchy::levels()[id.level()].tiles.Base(id.tileid());
+    // return midgard::PointLL(base_ll_.first, base_ll_.second);
```

Comment:
Why did this change?

Reply:
very insightful question! it has to do with getting full precision graph node lat lons. node lat lons are stored as an offset from the corner of the tile. for some reason we stored the corner of the tile in float in the tile header. the thing about a tile though is that we already know what the corner is by simple math on the tileid. so instead of storing it we just do a couple of multiplications here to get the corner of the tile in double precision rather than float that is stored. look  at the TODO i added at the very bottom of this file:

https://github.com/valhalla/valhalla/pull/2693/files#diff-02d2f61e72de41c5a1fb592eaf617f2e3c24c1882ed77791a4e876e9a76fd05eR600

Reply:
Ah nice. I wonder if we stored the corner ll in case we change tiling in the future and can no longer rely on the id? In anycase, sgtm 👍